packages/editor/src/components/document-outline/test/__snapshots__/index.js.snap

```diff
@@ -65,8 +65,8 @@ exports[`DocumentOutline header blocks present should render warnings for multip
       isValid={false}
       key="1"
       level="H1"
-      onClick={[Function]}
       path={Array []}
+      target="#block-12df094f-a677-4ab2-882d-1821a10307e5"
```

Comment:
Your test failures may be related? Need to update snapshot?

Reply:
updated snapshots